httpie/compat.py

```
@@ -0,0 +1,173 @@
+
+is_py2 = sys.version_info[0] == 2
+is_py26 = sys.version_info[:2] == (2, 6)
+is_py27 = sys.version_info[:2] == (2, 7)
+is_py3 = sys.version_info[0] == 3
+is_pypy = 'pypy' in sys.version.lower()
+is_windows = 'win32' in str(sys.platform).lower()
+
+
+if is_py2:
+    bytes = str
+    str = unicode
```

Ruff (0.8.2)

Undefined name `unicode`

(F821)